### PR TITLE
attendee/reg route update

### PR DESCRIPTION
**Pull request by jacobc2700**

### CODE REVIEW

File: src/services/events/events-router.ts [hunk 1]
```diff
@@ -5,6 +5,42 @@ import { Database } from "../../database";
...
+            const endTime: Date = event.endTime as Date;
+
+            if (startTime <= currentTime && endTime >= currentTime) {
+                currentEvent = event;
+                break;
+            } else if (startTime > currentTime) {
+                nextEvent = event;
+                break;
+            }
+        }
```
Comment (AydanPirani) [inline src/services/events/events-router.ts:28]: can this not all be cleaned up in one go: 

indOne({
    startTime: { $gte: now }
  }).sort({ startTime: 1 });

File: src/services/events/events-schema.ts [hunk 2]
```diff
@@ -2,6 +2,9 @@ import { Schema } from "mongoose";
 import { z } from "zod";
 import { v4 as uuidv4 } from "uuid";
 
+const events = ["A", "B", "C"] as const;
+export const EventType = z.enum(events);
```
Comment (AydanPirani) [inline src/services/events/events-schema.ts:6]:
```suggestion
export const EventType = z.enum(["A", "B", "C"]);
```

File: src/services/events/events-schema.ts [hunk 3]
```diff
@@ -53,4 +57,9 @@ export const EventSchema = new Schema({
         type: Boolean,
         default: false,
     },
+    eventType: {
+        type: String,
+        required: true,
+        enum: events,
```
Comment (AydanPirani) [inline src/services/events/events-schema.ts:63]: EventType.Values

**Review by AydanPirani — CHANGES_REQUESTED** [review]
a few changes, but lgtm otherwise!

**Review by AydanPirani — COMMENTED** [review]
lgtm! @divyack2  waiting on ur final approval

File: src/services/events/events-schema.ts [hunk 4]
```diff
@@ -2,6 +2,8 @@ import { Schema } from "mongoose";
 import { z } from "zod";
 import { v4 as uuidv4 } from "uuid";
 
+export const EventType = z.enum(["A", "B", "C"]);
```
Comment (divyack2) [inline src/services/events/events-schema.ts:5]: wait should this be named with like CORPORATE, SPEAKER, etc instead of A, B, C?

**Review by AydanPirani — APPROVED** [review]

**Pull request merged**